### ISSUE TITLE
Cache issue with firebase hosting

### ISSUE THREAD

**Issue by Xoce51** [isaachinman/next-i18next#809]
## Describe the bug
I'm using this module in a firebase project, since we have latency issues (our next server is executed in us-central1) we decide to use the firebase cache cdn [see doc](https://firebase.google.com/docs/hosting/manage-cache) and we managed to reduce from several seconds to several ms the page loading. 
Then we needed to add translation in our website, but after added this module we notice that the cache was not used anymore, I tried to remove the server i18n middleware and the cache was restored. I noticed that the server response was different when I added the middleware.
After some research I figure out that the response header was different (see screenshot below). I found a solution to fix it (I describe below) but created this issue to know if my solution could be improved.

## Occurs in next-i18next version
next-i18next: 4.5.0

## Steps to reproduce
 - Create a nextJS multilang app
 - Add server middleware
 - Deploy it on firebase server with cache setting enabled (with Cache-Control)

## Expected behaviour
The page loading should be the same with or without the module

## Screenshots
Server response header with cache and middleware disabled:
![with-cache](https://user-images.githubusercontent.com/5357297/91370237-7f146d80-e859-11ea-9fb8-849c71a11c5e.png)
Server response header without cache and middleware enabled
![without-cache](https://user-images.githubusercontent.com/5357297/91370255-8fc4e380-e859-11ea-87a1-c3b94274992c.png)

## OS (please complete the following information)
 - Firefox - unbutu 16

## Additional context
My solution, after noticed that the header were different I decided to remove manually the extra headers from the serveur
here is the code
```
  res.removeHeader('expires');
  res.removeHeader('accept-ranges');
  res.removeHeader('content-encoding');
  res.removeHeader('content-language');
  res.removeHeader('content-type');
  res.removeHeader('function-execution-id');
  res.removeHeader('server');
  res.removeHeader('set-cookie');
  res.set('Cache-Control', 'public, max-age=86400, s-maxage=86400');
```
But I'm still wondering if there is any option to do this or perhaps a PR can be done, if not hopes this will help someone

Than you

**Comment by isaachinman** [comment]
Please upgrade to the latest version of `next-i18next` and let me know if you still experience issues.

**Comment by Xoce51** [comment]
Hi, I updated the plugin to the version to the version 6 and removed the middleware because it seems it has been deleted.
But in my case I use express for my routing, so I installed `i18next-express-middleware` to make i18n works.

Is there any other way to use this module with express ?

Thank you

**Comment by isaachinman** [comment]
You can use `rewrites` for custom routes. Custom servers are no longer supported.

**Comment by isaachinman** [comment]
Hi there, I have just released `next-i18next@8.0.0`, which uses the internationalised routing built into NextJs v10+. This new version of `next-i18next` is quite different, thus making this PR no longer relevant. Thanks!
